test/Microsoft.TestPlatform.AcceptanceTests/BlameDataCollectorTests.cs

```diff
@@ -251,6 +251,44 @@ public void HangDumpChildProcesses(RunnerInfo runnerInfo)
         ValidateDump(2);
     }
 
+    [TestMethod]
+    [TestCategory("Windows-Review")]
+    [NetFullTargetFrameworkDataSource]
+    [NetCoreTargetFrameworkDataSource]
+    public void BlameDataCollectorAeDebuggerShouldCollectDump(RunnerInfo runnerInfo)
+    {
+        SetTestEnvironment(_testEnvironment, runnerInfo);
+
+        // Install AeDebugger
+        string dumpPath = Path.Combine(TempDirectory.Path, "Dumps");
+        Directory.CreateDirectory(dumpPath);
+
+        ExecuteVsTestConsole($"/AeDebugger:Install;ProcDumpToolDirectoryPath={_procDumpPath};DumpDirectoryPath={dumpPath}",
```

Comment:
Could this be done the same way as on blame where we find it in PATH or from env variable?

Reply:
Why?here we're installing procdump as postmortem debugger...we need all these information and are machine wide, I don't see benefit in use env var. I mean anyway I have to run this code before the real test. 
If you refer to the fact that we could pass parameter using env var, we could but for this first version(I don't know if we're missing something yet) I'd like to be really "simple" in the setup.

Reply:
I should have put more focus on the PATH part, that makes it easier to install procdump as a step before the test run (e.g. via chocolatey). And using the same procdump_path env variable would make it work automatically with vstest task which installs procdump and populates it.

On CI it is pretty hard to know what exact path you should specify.

Reply:
I can follow-up with the PATH support, I mean we can check the `ProcDumpToolDirectoryPath` and fallback to the PATH/env var, make sense.
I need more time for that and I'd like to ship something to let them test it, can we merge this part?